apps/shooting-clubs/controllers/confirm.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../common/controllers/confirm');
+
+module.exports = class ConfirmController extends Controller {
+
+  getFieldData(field, req) {
+    const defaults = super.getFieldData(field, req);
+
+    if (field.field === 'location-addresses') {
+      return defaults.value.map(address => {
+        address['address-category'] = [].concat(address['address-category']);
```

Comment:
but rather than actually mutating the value here, cant we just cast it below in the Object assign?

```js
return Object.assign({}, defaults, {
  label: address.address,
  value: _.castArray(address['address-category'])
    .map(category => req.translate(`fields.location-address-category.options.${category}.label`))
    .join('\n')
});
```